__maestro__/setup/onboarding.yml

```diff
@@ -0,0 +1,10 @@
+appId: ${APP_ID}
+---
+- tapOn: "WEITER"
+- tapOn: "WEITER"
+- tapOn: "WEITER"
+- tapOn: "WEITER"
+- tapOn: "WEITER"
+- tapOn: "WEITER"
+- tapOn: "WEITER"
+- tapOn: "Nein"
```

Comment:
in the future this needs to be dynamic per script, calling the onboarding settings from the server and creating the number of taps on "WEITER" and possible interactions with alerts

Reply:
Yes, that's right. But if you are going to use scripts for all apps, we will need much more customization options. Or do we only want to use it for the development version?

Reply:
Yes, that's right. But if you are going to use scripts for all apps, we will need much more customization options. Or do we only want to use it for the development version?

Reply:
we will see. but with the plans to get the settings from the server, it should work for others as well.